docs/maintainers/issue-management.md

```diff
@@ -8,6 +8,7 @@ Issues are opened for many different reasons.
 We use the following labels for issue classifications:
 
 * `Issue-Bug`: the issue is reporting a bug
+* `Issue-Code Cleanup`: the issue is for cleaning up the code with no impact on functionality
```

Comment:
Maybe remove the space in the label? This makes it easier to write manual filters in GitHub.

Please clarify - usually we have PRs with code/test cleanup not Issues.